sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/Models/ManagedInstance.cs

```diff
@@ -169,7 +169,7 @@ public ManagedInstance(string location, string id = default(string), string name
         /// instance.
         /// </summary>
         [JsonProperty(PropertyName = "identity")]
-        public ResourceIdentityWithUserAssignedIdentities Identity { get; set; }
+        public ResourceIdentity Identity { get; set; }
```

Comment:
This will result in a breaking change, please work with the breaking change team to get the associated PR signed off with Approved-BreakChange label

https://github.com/Azure/azure-rest-api-specs/pull/14439

Reply:
This change is actually fixing the breaking change that was earlier caused due to renaming this. This is just reversing that breaking change back to what it was before.
PR which introduced this change: https://github.com/Azure/azure-sdk-for-net/commit/939e15a7b89db3eed6b39a14b60387f22fb93ab1#diff-adc9a53c10c01948ba36af6ae1b8ef301f4802001380fb5e9867ab04e9192be7